### PR TITLE
breadcrumbs directive used in multiple pages

### PR DESCRIPTION
**Pull request by ahsu1230**

### CODE REVIEW

File: winterfell/webapp/src/templates/breadcrumbs.jade [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#vfi-breadcrumbs
+  div.breadcrumbs-wrapper
+    div.link-wrapper(ng-repeat="link in links")
+      span(ng-class="{'lighten': link.lighten}", ng-if="$index > 0") >
```
Comment (qwwqwwq) [inline winterfell/webapp/src/templates/breadcrumbs.jade]: Looks like theres a typo here

Comment (ahsu1230) [inline winterfell/webapp/src/templates/breadcrumbs.jade]: I want it to display a `>`. Maybe i should be using an html encoded character

Comment (qwwqwwq) [inline winterfell/webapp/src/templates/breadcrumbs.jade]: Oh i see, yeah not sure

**Comment by ahsu1230** [comment]
fixed typo

**Pull request merged**